src/replication.c

```diff
@@ -3511,11 +3527,19 @@ void syncWithPrimary(connection *conn) {
          *
          * EOF: supports EOF-style RDB transfer for diskless replication.
          * PSYNC2: supports PSYNC v2, so understands +CONTINUE <new repl ID>.
+         * BYPASS-CRC: supports skipping CRC calculations during full sync.
```

Comment:
BYPASS isn't the right word, since it implies we are doing something else instead. I think `SKIP-CRC` is what you want.